src/Expensify.tsx

```diff
@@ -236,6 +238,16 @@ function Expensify({
         Audio.setAudioModeAsync({playsInSilentModeIOS: true});
     }, []);
 
+    useEffect(() => {
```

Comment:
Wondering if we should run with with a `useLayoutEffect` so that we can run `Navigation.navigate` to route to the correct screen before the first paint.

Reply:
Done.
Tested again, working perfectly.